ztp/siteconfig-generator/siteConfig/siteConfig.go

```diff
@@ -2,8 +2,11 @@ package siteConfig
 
 import (
 	"fmt"
+	"path"
 	"reflect"
 	"strings"
+
+	"github.com/coreos/go-systemd/v22/unit"
```

Comment:
Is v22 the right version to use?  Does that correspond to the systemd version that's running under RHCOS 8?

Can you use whatever "latest" version was already in use?

Reply:
Updating to use latest